library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/SystemInformation.kt

```diff
@@ -6,4 +6,9 @@
 
 package com.datadog.android.sessionreplay.internal.recorder
 
-internal data class OrientationChanged(val width: Int, val height: Int)
+import android.content.res.Configuration
+
+internal data class SystemInformation(
+    val screenBounds: ViewGlobalBounds,
```

Comment:
maybe we need to rename `ViewGlobalBounds` class? Because here it is used in the context of the screen, and internally in the `ViewGlobalBounds` there is nothing specific for the view, it is just area description with starting point + width + height.

Reply:
yes...good point, I will just name it GlobalBounds